app/furniture/journal/keywords/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= keyword.canonical_keyword %></h1>
+<ul>
+<%= keyword&.aliases&.each do |aliass| %>
+  <li><%= aliass %></li>
```

Comment:
sssssssssss (thanks, Ruby)

Reply:
I was a/nnoyed/mused at the consequence of choosing `aliases` as a collection; and may wind up finding a different name to prevent the silliness.